Documentation/operations/troubleshooting.rst

```diff
@@ -713,6 +713,38 @@ When running in :ref:`arch_direct_routing` mode:
 
 4. Verify that the firewall on each node permits to route the endpoint IPs.
 
+Traffic to Pods of a Service is unbalanced
+------------------------------------------
+
+Symptom
+~~~~~~~
+
+Endpoint to endpoint communication between nodes is successful, but connections
```

Comment:
The following cases can become unbalanced:

1. A NodePort, LoadBalancer or externalIP service accessed from outside cluster when NodePort BPF is enabled.
2. A ClusterIP service is accessed within a cluster when the host reachables services feature is disabled.

In any case, it's required that the same client should recycle source ports. I'd suggest to rephrase the Symptom section by mentioning above and the recycle.